app/javascript/stylesheets/application.scss

```diff
@@ -1,9 +1,12 @@
 $govuk-assets-path: "~govuk-frontend/govuk/assets/";
 $govuk-font-family: "GDS Transport" , arial, sans-serif !important;
+$bg-orange: #FFA300 !important;
+
 @import "govuk-frontend/govuk/all";
 
 @import "colours";
 @import "links";
 
 @import "components/button";
 @import "components/header";
+@import "components/property-details.scss";
```

Comment:
Hey I think you can drop the `.scss` from the end of this as it's not needed. See the above import files ⬆️